tests/acceptance/pageObjects/FilesPageElement/appSideBar.js

```diff
@@ -23,6 +21,57 @@ module.exports = {
       }
       return this.api.page.FilesPageElement.filesList()
     },
+    selectTab: async function(tab) {
+      await this.waitForElementVisible(this.api.page.filesPage().elements.sideBar)
+      const panelVisible = await this.isPanelVisible(
+        tab,
+        this.api.globals.waitForNegativeConditionTimeout
+      )
+      if (panelVisible) {
+        return this
+      } else {
+        return this.useXpath()
+          .click(this.getXpathOfLinkToTabInSidePanel(tab))
+          .useCss()
+      }
+    },
+    /**
+     * return the complete xpath of the link to the specified tab in the side-bad
```

Comment:
```suggestion
     * return the complete xpath of the link to the specified tab in the side-bar
```
only if there are other changes to come anyway.

Reply:
fixed